tensorflow/python/ops/nn_grad.py

```diff
@@ -491,6 +491,19 @@ def _AvgPoolGrad(op, grad):
       data_format=op.get_attr("data_format"))
 
 
+@ops.RegisterGradient("AvgPoolV2")
+def _AvgPoolGradV2(op, grad):
```

Comment:
Thanks @yzhwang The PR has been updated with tests using placeholder now. Please take a look.